.flowconfig

```diff
@@ -10,6 +11,8 @@
 [include]
 
 [libs]
+./flow/interfaces/
```

Comment:
Agreed, but can we add another issue for that? I can fix that in next release and for users it is transparent. Right now I need to use this in my project for SSR and this blocks me with adoption